yarn.lock

```diff
@@ -2827,6 +2895,13 @@
   dependencies:
     "@react-types/shared" "^3.22.1"
 
+"@react-types/button@^3.9.4":
```

Comment:
looks like some duplicates again, you might try `npx yarn-deduplicate && yarn install`